sunpy/map/mapbase.py

```diff
@@ -1369,6 +1369,9 @@ def submap(self, bottom_left, top_right=None):
         Returns a submap of the map defined by the rectangle given by the
         ``[bottom_left, top_right]`` coordinates.
 
+        Any pixels which have at least part of their area inside the rectangle
+        are returned.
```

Comment:
Was this the behaviour before when using world coordinates?

Reply:
I think so